api/application.ts

```diff
@@ -53,35 +55,8 @@ app.use(errorStack)
 app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({extended: true}))
 
-// TODO: remove this when we have proper frontend configuration
```

Comment:
is this cookie no longer needed? just curious

Reply:
Discussed @ficklephil  and he said its not being used anywhere